ci-operator/config/openshift/machine-config-operator/openshift-machine-config-operator-master.yaml

```diff
@@ -65,18 +65,19 @@ tests:
   container:
     from: src
 - as: e2e-aws
-  skip_if_only_changed: ^docs/
+  skip_if_only_changed: ^docs/|\.md$|^(?:.*/)?(?:\.gitignore|OWNERS|OWNERS_ALIASES|LICENSE)$
```

Comment:
Not sure if this would apply to yaml configs like this, but if this were code I was wondering if it would be clearer to have a single commented variable explaining the regex and then reuse it everywhere

Reply:
Eg store it in `skipDocs` and then `<<: *skipDocs` everywhere

Reply:
I don't know, some of things release repo does things are mystery to me. I would think if we can do that other projects may have already adopted it.

Reply:
I had to look that up, that's some YAML sorcery.  It looks like `<<:` is a more complex version of [an anchor](https://yaml.org/spec/1.2.2/#alias-nodes)?  (Also in TIL, there's a new version of the YAML spec after 12 years, like your not really favorite band exactly but one that is frequently on the radio and you mostly tolerate grudgingly getting back together)

Offhand if that works, seems nicer sure. 

OTOH I think it'd be even better to have ci-operator have a default regex that can be overridden/appended to for all repos.

Reply:
`<<` is a merge key https://yaml.org/type/merge.html (that doc says it's a draft so not sure what supports it)

Reply:
I would leave things the way it is. If it supports adding variable, there are multiple places we can use it in this repo.